athenet/sparsifying/utils/interval.py

```diff
@@ -0,0 +1,364 @@
+"""Intervals in Theano including special functions for sparsifying."""
```

Comment:
"Intervals in Theano" sounds like there are some intervals in theano ans you are expanding it